pkg/metrics/parser.go

```diff
@@ -84,6 +84,12 @@ func toInt64(i interface{}) int64 {
 		return v
 	case time.Duration:
 		return int64(v)
+	case string:
+		value, err := strconv.ParseInt(v, 10, 64)
+		if err != nil {
+			panic(err)
```

Comment:
I'm not confident that we want to panic here. It's user supplied data after all. So a single malformed MQTT message will bring the exporter into a crashloop? :thinking: